CRM/Member/Import/Parser/Membership.php

```diff
@@ -670,7 +670,7 @@ public function membership_format_params($params, &$values, $create = FALSE) {
           break;
 
         case 'membership_type_id':
-          if (!CRM_Utils_Array::value($value, CRM_Member_PseudoConstant::membershipType())) {
+          if (empty(CRM_Member_PseudoConstant::membershipType([$value]))) {
```

Comment:
This seems wrong. Test failure too.

Reply:
Fixed using `array_key_exists()` which is what it was doing.